src/tasks/AutoBidAuctionTask.py

```
@@ -0,0 +1,401 @@
+        self._check_stop()
+        box_emote_btn = self.box_of_screen_scaled(1920, 1080, 58, 972, width_original=27, height_original=27)
+        box_first_emote = self.box_of_screen_scaled(1920, 1080, 237, 533, width_original=65, height_original=55)
+        self.log_info("准备发送表情包")
+        self._check_stop()
+        self.operate_click(box_emote_btn, after_sleep=0.8)
+        self.sleep(0.6)
+        self._check_stop()
+        self.operate_click(box_first_emote, after_sleep=0.5)
+        self.log_info("表情包发送完成")
+
+    def _input_fixed_price(self, price: int = None):
```

Ruff (0.16.1)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)